test/Microsoft.AspNetCore.JsonPatch.Test/Dynamic/DynamicObjectAdapterTests.cs

```diff
@@ -0,0 +1,1215 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.AspNetCore.JsonPatch.Exceptions;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNetCore.JsonPatch.Test.Dynamic
+{
+    public class DynamicObjectAdapterTests
```

Comment:
I'm sending one of my minions to help you sort out this mess later today. Don't fret, help is on the way